boards/pimoroni-plasma-2040/examples/pimoroni_plasma_2040_blinky.rs

```diff
@@ -0,0 +1,71 @@
+//! Blinks the 3 colour LEDs on a Pimoroni Plasma 2040 in sequence
+#![no_std]
+#![no_main]
+
+use cortex_m_rt::entry;
+use defmt::*;
+use defmt_rtt as _;
+use embedded_hal::digital::v2::OutputPin;
+use embedded_time::fixed_point::FixedPoint;
+use panic_halt as _;
+
+use pimoroni_plasma_2040 as bsp;
+
+use bsp::hal::{
+    clocks::{init_clocks_and_plls, Clock},
+    pac,
+    sio::Sio,
+    watchdog::Watchdog,
+};
+
+#[entry]
+fn main() -> ! {
+    info!("Program start");
+    let mut pac = pac::Peripherals::take().unwrap();
+    let core = pac::CorePeripherals::take().unwrap();
+    let mut watchdog = Watchdog::new(pac.WATCHDOG);
+    let sio = Sio::new(pac.SIO);
+
+    // External high-speed crystal on the pico board is 12Mhz
```

Comment:
Remove this comment. Board is not a pico, this isn't a HAL where user needs to care about crystal speed - that's what XOSC_CRYSTAL_FREQ is for
```suggestion
```

Reply:
Removed for Plasma 2040 example and in the example from whence it came.